test/pages/ReporterTypePageSpec.scala

```diff
@@ -0,0 +1,164 @@
+                .withPage(IndWhereDoYouLivePage, booleanField)
+                .withPage(IndWhatIsYourPostcodePage, postcode)
+                .withPage(AddressLookupPage, Seq(addressLookup))
+
+              val result = ReporterTypePage.cleanup(Some(LimitedCompany), ua).success.value
+
+              result.get(IndWhatIsYourNINumberPage) should (be(None) or not be empty)
+              result.get(IndContactNamePage) should (be(None) or not be empty)
+              result.get(IndDateOfBirthPage) should (be(None) or not be empty)
+              result.get(RegistrationInfoPage) should (be(None) or not be empty)
+              result.get(IndWhatIsYourNamePage) should (be(None) or not be empty)
+              result.get(DateOfBirthWithoutIdPage) should (be(None) or not be empty)
+              result.get(IndWhereDoYouLivePage) should (be(None) or not be empty)
+              result.get(IndWhatIsYourPostcodePage) should (be(None) or not be empty)
+              result.get(AddressLookupPage) should (be(None) or not be empty)
+          }
+        }
+      }
+      "must clear answers" - {
+        "when answer changes to anything other than 'An individual not connected to a business' or 'Sole Trader'" in {
+          forAll(testParamGenerator) {
+            case (addressLookup, address, postcode, name, booleanField, nino, registrationInfo, dob, stringField, utr) =>
+              val ua = emptyUserAnswers
+                .withPage(IndWhatIsYourNINumberPage, nino)
+                .withPage(IndContactNamePage, name)
+                .withPage(IndDateOfBirthPage, dob)
+                .withPage(RegistrationInfoPage, registrationInfo)
+                .withPage(IndWhatIsYourNamePage, name)
+                .withPage(DateOfBirthWithoutIdPage, dob)
+                .withPage(IndWhereDoYouLivePage, booleanField)
+                .withPage(IndWhatIsYourPostcodePage, postcode)
+                .withPage(AddressLookupPage, Seq(addressLookup))
+                .withPage(IndSelectAddressPage, stringField)
+                .withPage(IndSelectedAddressLookupPage, addressLookup)
+                .withPage(IsThisYourAddressPage, booleanField)
+                .withPage(IndUKAddressWithoutIdPage, address)
+                .withPage(IndNonUKAddressWithoutIdPage, address)
+                .withPage(IndContactEmailPage, stringField)
+                .withPage(IndContactHavePhonePage, booleanField)
+                .withPage(IndContactPhonePage, stringField)
+                .withPage(IndDoYouHaveNINumberPage, booleanField)
+
+              val result = ReporterTypePage.cleanup(Some(LimitedCompany), ua).success.value
+
+              result.get(IndWhatIsYourNINumberPage) mustBe empty
+              result.get(IndContactNamePage) mustBe empty
+              result.get(IndDateOfBirthPage) mustBe empty
+              result.get(RegistrationInfoPage) mustBe empty
+              result.get(IndWhatIsYourNamePage) mustBe empty
```

Comment:
Pet peeve but the test above uses `should` matchers and this one uses `must` matchers. Please can you standardise on just one?